sources/Content_Quality/missing_rambam_intros.py

```diff
@@ -0,0 +1,50 @@
+import django
+
+django.setup()
+
+from sefaria.model import *
+from sefaria.helper.schema import convert_simple_index_to_complex, convert_jagged_array_to_schema_with_default, \
+    insert_first_child
+
+
+def convert_children_to_schema_nodes(tref):
+    book_title = Ref(tref).index.title
+    i = library.get_index(book_title)
+    # Convert to Schema nodes
+    for ja_node in i.nodes.children:
+        if isinstance(ja_node, JaggedArrayNode):
+            convert_jagged_array_to_schema_with_default(ja_node)
+
+
+def create_intro():
+    intro = JaggedArrayNode()
+    intro.add_structure(["Paragraph"])
+    intro.add_shared_term("Introduction")
+    intro.key = "Introduction"
+    intro.validate()
+    return intro
+
+
+def create_intro_complex_text(node):
+    intro = create_intro()
+    insert_first_child(intro, node)
+
+
+def convert_index(node):
+    index_title = node.title
+    convert_simple_index_to_complex(library.get_index(index_title))
+    create_intro_complex_text(Ref(index_title).index_node)
```

Comment:
node.nodes is better

Reply:
Here it's a funny case - @nsantacruz and I dealt with it last summer when working on the generalized intro scripts. For some reason, we need to force it to read the updated index, and the best way to do that is via the `Ref` call. I agree though that it's messy and inconsistent, and would love to find a better way of doing it. I had tried cleaning up this part of the code a few other ways / but this was the only way that managed to get it to read the new schema for the Index.